src/open_inwoner/scss/components/PluginCard/PluginCard.scss

```diff
@@ -0,0 +1,54 @@
+.plugin-card {
```

Comment:
This seems specific to Userfeed? We do have multiple plugin-cards, but apparently those do not need this styling...?

Reply:
I thought I'd make this generic, because the appointments plugin cards have (almost) the same styling and I didn't want to duplicate it:

![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/dd33f328-7bf5-4f7d-87ec-aa20d545fd65)

Reply:
OK - seems like the appointments don't need a notification color (yet) so approving this now.